javascript/README.md

```diff
@@ -0,0 +1,16 @@
+# Title: RESTy
```

Comment:
I don't think this belongs here🤔

Reply:
I dont know how that happened.  I will fix that